api/setup.ts

```diff
@@ -0,0 +1,65 @@
+//Imports
+import { parse,  stringify } from "https://deno.land/std@0.97.0/encoding/yaml.ts";
+import { DOMParser, Element } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
+
+//Load static/base.html
+let html: any = new DOMParser().parseFromString(await Deno.readTextFile("./static/base.html"), "text/html")!
+
+//Load config.yml
+let config: any = parse(await Deno.readTextFile("./config.yml"))
+
+//Set company name
+if(config["name"]){
+    //TODO
+}
+
+//Set custom css
+config["styles"]!.forEach((stylePath: string ) => {
```

Comment:
```suggestion
config["styles"]?.forEach((stylePath: string ) => {
```
Is that what you're looking for ? The [optional chaining operator](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to access possibly `undefined` or `null` values ?

Reply:
Correct